ref(docs): Update README with more info on prerequisites (#234)
getsentry/action-release: README.md

> A new option to inject Debug IDs into source files and sourcemaps was added to the action to ensure proper un-minifaction of your stacktraces. We **strongly recommend enabling** this by setting `inject: true` in your action alongside providing a path to sourcemaps.

Reviewer: ```suggestion
A new option to inject [Debug IDs](https://docs.sentry.io/platforms/javascript/sourcemaps/troubleshooting_js/artifact-bundles/) into source files and sourcemaps was added to the action to ensure proper un-minifaction of your stacktraces. We **strongly recommend enabling** this by setting `inject: true` in your action alongside providing a path to sourcemaps.
```

This allows us to completely delete the following line (see next suggestion)

Author: This message has already gone out with the release last week, I'd like to keep them in sync.

Reviewer: What release last week, could you link it?

Author: https://github.com/getsentry/action-release/releases

Reviewer: @andreiborza this does not change the message at all, it just makes the "Debug IDs" text link to the Debug ID page, so we can delete the line afterwards

> [Learn more about debug ids](https://docs.sentry.io/platforms/javascript/sourcemaps/troubleshooting_js/artifact-bundles/)

Reviewer: ```suggestion
```

Please refer to the [release page](https://github.com/getsentry/action-release/releases) for the latest release notes.

## Prerequisites

### Create an Organization Auth Token

For this action to communicate securely with Sentry, you'll need to [create an organization auth token](https://docs.sentry.io/account/auth-tokens/#organization-auth-tokens).

> Alternatively, you can also use a [User Auth Token](https://docs.sentry.io/account/auth-tokens/#user-auth-tokens), with the "Project: Read & Write" and "Release: Admin" permissions.

Author: Thanks, Steven gave the same feedback on the docs PR. 👍

> You also need to set your Organization and Project slugs and if you're using a self-hosted Sentry instance, provide the URL used to connect to Sentry via SENTRY_URL.

Reviewer: The organization and Sentry URL are encoded in the org auth token. We recommend that users do not manually set these when they are using an org auth token. Users should only need to set the project slug.

```suggestion
You also need to set your project slug.
```

Author: Does that hold true for self-hosted too?

Anyway, this isn't a simple docs/readme change as the action has code that depends on these. We can think of removing in another issue. Let's discuss this in our meeting.

Reviewer: Yes, it applies to self hosted (minus, some old self-hosted versions that did not encode the URL due to a bug). Sentry CLI (at least for the URL, not sure about the org) will ignore the manually provided values and use the ones encoded in the token, and it also issues a warning in case of a difference.

The whole point of encoding the URL and org in the token was to eliminate the need to manually specify them, so we should definitely remove any code that depends on these being manually provided from the GitHub action

Author: As discussed, let's tackle this in https://github.com/getsentry/action-release/issues/212

Reviewer: #212 seems to imply we would need to make these changes in a major release, but I think we can make them in a minor release, too

Author: We can, but priority wise we want to tackle a new major first.

```bash

> SENTRY_AUTH_TOKEN=sntrys_YOUR_TOKEN_HERE
> SENTRY_ORG=example-org
> SENTRY_PROJECT=example-project
> # For self-hosted
> # SENTRY_URL=https://my-sentry-url

Reviewer: Remove the items as mentioned above. Also, small nit would be just to have `SENTRY_AUTH_TOKEN=YOUR_TOKEN_HERE` because `sntrys_` is part of the token (or we could do `sntrys_...` instead)

```suggestion
SENTRY_AUTH_TOKEN=YOUR_TOKEN_HERE
SENTRY_PROJECT=example-project
```

Author: The `sntrys_YOUR_TOKEN_HERE` is what our docs use when users are not logged in, I'd like to keep it that way.

As for the others, see the reply above regarding the action code depending on them.
